.idea/compiler.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <option name="DEFAULT_COMPILER" value="error-prone" />
```

Comment:
don't commit this; to prevent this from happening, install the error-prone compile plugin in IntelliJ and then make sure you set the java compiler (in IJ settings) to the error prone compiler